android/app/src/main/java/org/fossasia/openevent/activities/SearchActivity.java

```diff
@@ -127,102 +118,32 @@ public void handleIntent(Intent intent) {
         searchQuery(query);
     }
 
-    private void searchQuery(String constraint) {
-        results.clear();
-
-        if (!TextUtils.isEmpty(constraint)) {
-            String query = constraint.toLowerCase(Locale.getDefault());
-            String wildcardQuery = String.format("*%s*", query);
-            addResultsFromTracks(wildcardQuery);
-            addResultFromSessions(wildcardQuery);
-            addResultsFromSpeakers(wildcardQuery);
-            addResultsFromLocations(wildcardQuery);
-        }
+    @Override
+    public void onSaveInstanceState(Bundle bundle) {
```

Comment:
Please delete this method now. Since the only thing you're doing is calling super, there is no need for this method.

Reply:
@srv-twry Okay

Reply:
@srv-twry But there can be future uses of this method right?

Reply:
This is an overridden method, which if not called explicitly for the SearchActivity will automatically be called for the super class. In case there are future requirements , one can easily create it again.

It is mainly used to save states, when orientation changes. Now that we have ViewModels which survive orientation changes, i don't think we will need it. In case we need it , we will create it.